### ISSUE TITLE
new files not showing up after running jbang edit

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang#750]
have a x.java, run jbang edit x.java
have x.java have `//SOURCES *.java`

`touch Other.java`

run `jbang edit x.java`

IDE or at least filesystem should now have symbolic link for the file.

**Comment by mbechto** [comment]
I have experienced the same with a sources folder `//SOURCES src`.
Would be really nice if it worked, currently I am copying files from my temporary IntelliJ workspace back into the source dir 😃

**Comment by maxandersen** [comment]
Have you tried using the ide plugin directly so you dont need symbolic link tree at all?

**Comment by mbechto** [comment]
> Have you tried using the ide plugin directly so you dont need symbolic link tree at all?

Good point, just tried that. Maybe I am doing something wrong, but I can't make it work:

When I open a JBang script file, the plugin shows me the script name & dependency list in a tool window on the right (all my `//DEPS`). I can also run the script using the IntelliJ run line action. But when I start to type code, auto-completion is not showing anything ("No suggestions"). Placing the cursor on a type from a dependency e.g. I type `YamlPath.` it's as if the type does not exist.

This works just fine with `jbang edit` since it creates a Gradle project that IntelliJ understands and can derive the classpath from I guess.

**Comment by maxandersen** [comment]
Yeah. Intellij is not being as automatic as vscode and eclipse. Right click In the source editor and click synchronize jbang. Then things should work.  

See https://github.com/jbangdev/jbang-idea/issues/102 and add your voice here :)
